locales/bn/bn.json

```diff
@@ -400,22 +400,22 @@
     "October": "অক্টোবর",
     "of": "সর্বমোট",
     "Oh no": "ওহ না",
-    "Oman": "মলডোভা",
+    "Oman": "ওমান",
     "Once a team is deleted, all of its resources and data will be permanently deleted. Before deleting this team, please download any data or information regarding this team that you wish to retain.": "একটি দল মুছে ফেলা হয় একবার, তার সম্পদ ও তথ্য সব স্থায়ীভাবে মুছে ফেলা হবে. এই দল মুছে ফেলার পূর্বে, আপনি বজায় রাখতে চান যে এই দল সংক্রান্ত কোনো তথ্য বা তথ্য ডাউনলোড করুন.",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`), but here - a period (`.`).

This translation was by machine.